datana-gmbh/datapool-api/1.0/config/packages/datana_datapool_api.yaml

```diff
@@ -0,0 +1,30 @@
+services:
+    _defaults:
+        autowire: true
+
+    Datana\Datapool\Api\DatapoolClient:
+        arguments:
+            - '%env(DATAPOOL_API_BASE_URI)%'
+            - '%env(DATAPOOL_API_USERNAME)%'
+            - '%env(DATAPOOL_API_PASSWORD)%'
+
+    Datana\Datapool\Api\AktenEventLogApi: ~
+    Datana\Datapool\Api\NullAktenEventLogApi: ~
+
+    Datana\Datapool\Api\AktenzeichenApi: ~
+    Datana\Datapool\Api\FakeAktenzeichenApi: ~
+
+when@prod:
```

Comment:
This is not needed. 

Currently there is no service for `Datana\Datapool\Api\AktenzeichenApiInterface` defined by default. I suggest to make "prod" environment the default one.

Reply:
No, in this case if someone would add a `stage` environment, it is not matched by `when@dev` and `when@test`, so the result would be, that the prod one is used, which could end up in write operations to production system of the service.... Thats what I want to avoid!